.github/workflows/crawler.yml

```diff
@@ -163,9 +202,21 @@ jobs:
       run: |
         echo "Crawler is working. Proceed to text processing pipeline."
 
+  milestone-2-pipeline:
+    name: Starting pipeline checks!
+    needs: [ milestone-1-crawler-is-working ]
+    runs-on: ubuntu-latest
+    timeout-minutes: 1
+    if: ${{ ! contains(github.event.head_commit.message, '[skip-pipe]') }}
```

Comment:
so the condition here should be different

Reply:
fixed